### PR TITLE
Added a route for editing a teacher's information from the admin UI

### LINKED ISSUE #62: Edit teacher profiles from teacher overview
Add the ability to edit a teacher profile from the teacher overview using the "edit"-button.

### PR DESCRIPTION
**Pull request by hugolundin**
Closes #62 

Added the ability to change a teacher's first name, last name or email adress.
- Added EditUserForm()
- Added a edit_teacher.html-template.
- Added a route in views.py for changing teacher attributes by changing the user object.

### CODE REVIEW

**Comment by Greenheart** [comment]
Tested locally and working pretty nice!

**One question:** Do we want admins to **always** have to verify the email even though they don't try to change it? 

How about filling both fields with the current email for the teacher?

If we want, a small JS could be added to automatically empty the email fields if any of them get a changed value

File: web/app/admin/views.py [hunk 1]
```diff
@@ -53,6 +53,21 @@ def add_teacher():
     return render_template('admin/add.html', form=form)
 
 
+@admin.route('/teachers/edit/<int:id>', methods=['GET', 'POST'])
+@login_required
+@admin_required
+def edit_teacher(id):
+    user = User.query.filter_by(id=id).first()
+    form = EditTeacherForm(obj=user)
```
Comment (Greenheart) [inline web/app/admin/views.py:61]: I guess this is the line that populate the form with the current data. Where should we force the form into displaying the user's email twice?

**Comment by Limpan** [comment]
Why require the user/admin to write the email twice? Aren't they capable humans? Better send a confirmation email, at least if it's the user! 

If it's admin that changes, he/she can always change it again or be careful when entering the email. Maybe send some kind of "hey, this admin dude changed your email, hope it's working"-message.

Best way to validate emails is always to try, double entering just makes the sloppy users copy paste and annoys the prudent users.

**Comment by Greenheart** [comment]
> Maybe send some kind of "hey, this admin dude changed your email, hope it's working"-message.

Confirmation email seems like a better idea!

**Comment by shhmon** [comment]
Also, it might be a good idea to use [keywords for closing issues](https://help.github.com/articles/closing-issues-via-commit-messages/) when adding commits or "pull requesting" with something that fixes an issue. This pull request could for example automatically close #62.

Correct me if I'm wrong

**Comment by hugolundin** [comment]
@limpan @Greenheart I did not actually do this intentional haha. My thought was to only have it once, but I guess my tired brain did not want to do the same thing. I will fix it asap.

@catagen thanks for the tip. Actually had no idea that this existed.

**Comment by Limpan** [comment]
> Confirmation email seems like a better idea! @Greenheart 

I'm not so sure. I think the situation is quite different. Whether the admin or the user initiates the email change does matter, in my mind.

Users should probably not be able to change their email by themselves anyway...

**Pull request merged**